### DISCUSSION TITLE
GitHub Student Developer Pack - github copilot

### DISCUSSION THREAD

**Discussion by studentug** [community/community#32199] [GitHub Education] [answered]
Hi!
I've recently received GitHub Student Developer Pack but I don't see Github Copilot available. As I can see, other students have this feature but don't see any in my account. Is it restricted or do I have to satisfy additional conditions?

I am looking forward to the answer!

**Comment by studentug** [comment, marked as answer]
That was fast. I had to go to the Github Coplilot page directly, and from there, activate the license. But It works to mention, that on this page https://education.github.com/pack/offers?sort=popularity&tag=Developer+tools there is no information that it's available.

**Reply by wsauthoff** [reply]
First you need to be a verified student through GitHub's Global Campus. Then you need to signup for GitHub Copilot while signed into your GitHub account as a verified student.

1. [Link to instructions](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#applying-to-github-global-campus) on how to apply for GitHub Global Campus that includes free GitHub Copilot access through the Student Developer Pack. 
2. [Link to GitHub Copilot signup page](https://github.com/github-copilot/signup) that will recognizes your GitHub account having a Student Developer Pack.

**Comment by ali-saranj** [comment]
چرا عکسم در قسمتی که می‌خواهم بگم دانش آموز هستم آپلود نمی شود و همین طور مینویسه درحال آپلود

**Comment by ali-saranj** [comment]
Why do I not upload my photo in the part I want to say

**Comment by diva-21** [comment]
hey @studentug ,
Github Copilot is a separate package bcz it has access and supports for different members like for Student it will be free, for educators and proffesionals it will be of a different kind. So you have to go to the official website of Github Copilot and link your github acc which has developer pack and make sure you tick Student option. Then open your editor and install the extension and then again login that with ur Github acc, then you can activate Copilot

This might help you
https://docs.github.com/en/copilot/getting-started-with-github-copilot

**Comment by payal-t** [comment]
Hey @studentug 
Facing an issue while logging in to GitHub with the student developer pack,

![image](https://user-images.githubusercontent.com/71988321/200504898-602721fa-c2ac-4fc4-9fb4-8e6cc16564f6.png)

Please check .

**Reply by diva-21** [reply]
As its showing , can you please once confirm with your school department because many schools are registered with Github

**Reply by payal-t** [reply]
Actually am not getting the mail from GitHub for an activation link.Please check

**Reply by samus-aran** [reply]
Hi @payal-t 
The notification on your application form states that you need to contact your school's GitHub administrator to receive further instructions. Your particular school verifies and provides their students the activation link, not GitHub.  👍

**Reply by wsauthoff** [reply]
Using a .edu email address may solve your issue (not a gmail).

**Comment by Raajheer1** [comment]
Hello,

I have a verified Github Student Account (last verified as a student on Jan 2, 2023), however I no longer have access to Github Copilot, even though I had access previously (through Technical Preview).

<img width="548" alt="image" src="https://user-images.githubusercontent.com/41763998/211262979-c40d503e-b111-41f5-85a5-d833bf2353b6.png">
I have attempted to enroll in CoPilot again however my options are as seen above. It doesn't appear to detect that I have a student account and is not offering a free option.

Any ideas on anything else I can try?

**Reply by bartekpacia** [reply]
Same here. Today morning my Copilot license from GitHub Student Pack apparently stopped working and I no longer can use Copilot.

**Reply by gauravmehta13** [reply]
Same here, Although now received another email 
![image](https://user-images.githubusercontent.com/51997260/211276352-793abcad-2a66-41da-b776-acd64b54e4aa.png)

**Reply by Raajheer1** [reply]
Hello,

I received that very same email and everything seems to be back in working order for me.

**Reply by KNOXZhao** [reply]
same 
I wonder if they change the policy (say student's pack no longer contains copilot)

**Comment by diva-21** [comment]
same here

**Comment by Huntca10** [comment]
same here

**Comment by ivanparramartinez** [comment]
Same here. Copilot stopped working like an hour ago and I don't know how to re-enable it.

**Comment by Jianhua-Wang** [comment]
same here

**Comment by betty2310** [comment]
Same here, wonder if they change the policy?

**Comment by inFngNam** [comment]
Same here

**Comment by betty2310** [comment]
Pls check this topic https://github.com/orgs/community/discussions/43634

**Comment by Telenergy** [comment]
Does someone found any suggestions on this issue? I'm so bad at all these technologies so I can't fix it by myself... All I use for education from the web is GitHub and a source with writing help (a great [essay on fences](https://happyessays.com/free-essays/fences/) was the last one I found there). Digitalizing education makes sense, but after years of book school studies that's pretty hard to adapt. I think it sounds like I'm a dino, but I'm just from a conservative family :)

**Reply by betty2310** [reply]
Hi @Telenergy, I try to reinstall it and it works now, may be you should.

**Comment by wsauthoff** [comment]
First you need to be a verified student through GitHub's Global Campus. Then you need to signup for GitHub Copilot while signed into your GitHub account as a verified student.

1. [Link to instructions](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#applying-to-github-global-campus) on how to apply for GitHub Global Campus that includes free GitHub Copilot access through the Student Developer Pack. 
2. [Link to GitHub Copilot signup page](https://github.com/github-copilot/signup) that will recognizes your GitHub account having a Student Developer Pack.

**Reply by nehSgnaiL** [reply]
Thx, the second link successfully recognised my education benefits.

**Comment by mvosiljonov** [comment]
Hi, i have registered and my uni is approved, but still i cant use github copilot on vs code. 
![image](https://github.com/community/community/assets/113694351/a749d63f-5c78-4e77-809b-e471f18d892c)
![image](https://github.com/community/community/assets/113694351/eb0bd00b-0965-4136-a20f-851894813fed)
I donno smth i'm doing wrongly or not

**Reply by mvosiljonov** [reply]
@studentug

**Comment by btarg** [comment]
I'm also unable to access my student benefits over a month after applying, waiting on a support ticket right now.
